ci.cue

```diff
@@ -110,8 +114,10 @@ dagger.#Plan & {
 					]
 				}
 				env: {
-					DAGGER_BINARY:     "/src/dagger"
-					DAGGER_LOG_FORMAT: client.env.DAGGER_LOG_FORMAT
+					DAGGER_BINARY:      "/src/dagger"
+					DAGGER_LOG_FORMAT:  client.env.DAGGER_LOG_FORMAT
+					DAGGER_CACHE_FROM?: client.env.DAGGER_CACHE_FROM
+					DAGGER_CACHE_TO?:   client.env.DAGGER_CACHE_TO
```

Comment:
I understand those changes but it will miss `GITHUB` environment to works in GitHub Actions no?

Reply:
Doh 🤦 Thanks, good catch, fixed that.

It's ironic because I was working on ways to improve our visibility into cache hits+misses during CI runs when I noticed this issue and figured I'd fix it up quickly.